src/components/vuestic-mixins/SelectableList/SelectableListMixin.spec.js

```diff
@@ -0,0 +1,94 @@
+import { shallowMount } from '@vue/test-utils'
+import { SelectableListMixin } from './SelectableListMixin'
+
+describe('SelectableListMixin', () => {
+  let baseComponentOptions
+
+  const prepareWrapper = (prop, value) => {
+    baseComponentOptions.propsData[prop] = value
+    return shallowMount({}, baseComponentOptions)
+  }
+
+  beforeEach(() => {
+    baseComponentOptions = {
+      render: () => '',
+      mixins: [SelectableListMixin],
+      propsData: {},
+    }
+  })
+
+  it('should mount without erros', () => {
+    const wrapper = shallowMount({}, baseComponentOptions)
+    expect(wrapper).toBeDefined()
+    expect(wrapper.props().options).toEqual([])
+    expect(wrapper.props().textBy).toEqual('text')
+    expect(wrapper.props().valueBy).toEqual('value')
+    expect(wrapper.props().trackBy).toEqual('value')
+    expect(wrapper.props().disabledBy).toEqual('disabled')
+    expect(wrapper.props().outputObject).toBeFalsy()
+  })
+
+  describe('getValue', () => {
+    it('should work with a string', () => {
+      const wrapper = prepareWrapper('valueBy', 'testValue')
```

Comment:
Maybe just use destructuring instead? You won't need `beforeEach` in that case.
```
prepareWrapper({valueBy: 'testValue'})
```
Also, base component options it might make sense to define once before `describe`.